cmd/tcl/testworkflow-toolkit/commands/artifacts.go

```diff
@@ -94,18 +97,24 @@ func NewArtifactsCmd() *cobra.Command {
 				}
 			}
 
+			var handlerOpts []artifacts.HandlerOpts
 			// Archive
 			if env.CloudEnabled() {
+				ctx, cancel := context.WithTimeout(cmd.Context(), 30*time.Second)
+				defer cancel()
+				client := env.Cloud(ctx)
+				defer client.Close()
+				handlerOpts = append(handlerOpts, artifacts.WithPostProcessor(artifacts.NewJUnitPostProcessor(filesystem.NewOSFileSystem(), client)))
```

Comment:
Shouldn't this be optional? Otherwise, when the Agent is updated, but the Cloud API is not, the step will break (as postprocessor will fail)

Reply:
that makes sense